### DISCUSSION TITLE
Jamulus stops Orca screen reader speaking on Mint

### DISCUSSION THREAD

**Discussion by a-singer** [jamulussoftware/jamulus#3314] [Bug Investigation]
I may say I am new to the GUI in Linux generally and to Mint in particular. I'm not sure what's happening, so this may well be user error. To reproduce, install Jamulus on Mint with the mate desktop. I did this through apt, perhaps I should have done it differently. Press alt+meta+s to start the ORCA screen reader. Launch Jamulus from the alt+f1 menu. Orca just stops speaking. The only way to get it to start again is to hit alt+f4 a few times and get out of Jamulus. Again, I'm not sure what's happening here, it may be Orca is failing or it may be that Jamulus is doing something weird with sound. I lose all access to the screen, and though I do have a braille display, it's not in front of me at the moment, so I'm not sure what's going on. I would appreciate it if someone could have a look and see what's happening on the screen or if this is a known issue.

**Comment by ann0see** [comment]
> Jamulus is doing something weird with sound

This can very well be an issue. Orca probably conflicts with JACK to some extent.

I'd select another output for the screen reader than the instrument/microphone + headphone.

**Comment by ann0see** [comment]
@chigkim may know more since he's experienced with accessibility.

**Comment by ignotus666** [comment]
@a-singer as @ann0see says, it's because JACK takes over the audio device. You need to install `pulseaudio-module-jack`, which exposes your computer's system audio to JACK and turns it into another in/out device. You may have to reboot or log out/in again. I just tested it (I also use Mint Mate) and it works for me, though I did have to manually connect Pulseaudio's output to the system input in QjackCtl.
